site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -307,12 +372,17 @@ const ProgramRequirementsList: FC<RequireCourseListProps> = ({ requirements, sto
     .flatMap((quarter) => quarter.courses)
     .map((course) => [course.id, course.minUnits]);
   const transferCourseMap = transferredCourses.map((t) => [t.courseName.replace(/\s/g, ''), t.units ?? 0]);
+  const transferCourseMapWithType = transferredCourses.map((t) => [
+    t.courseName.replace(/\s/g, ''),
+    { units: t.units ?? 0, transferType: t.transferType },
+  ]);
 
   const takenCourseSet: CompletedCourseSet = Object.assign(
     {},
     Object.fromEntries(roadmapCourseMap),
     Object.fromEntries(transferCourseMap),
   );
+  const transferredCourseSet: CompletedCourseSetWithType = Object.fromEntries(transferCourseMapWithType);
```

Comment:
As per previous comment, just use the same variable

```suggestion
  const takenCourseSet: CompletedCourseSetWithType = Object.assign(
    {},
    Object.fromEntries(roadmapCourseMap),
    Object.fromEntries(transferCourseMap),
  );
```